lib/pymedphys/_experimental/streamlit/apps/collimator_corrections.py

```diff
@@ -39,3 +46,277 @@ def main():
     except FileNotFoundError:
         st.error("Winston Lutz results not yet calculated/saved for this date.")
         st.stop()
+
+    dataframe["x_centre"] = np.mean(dataframe[["x_lower", "x_upper"]], axis=1)
+    dataframe["y_centre"] = np.mean(dataframe[["y_lower", "y_upper"]], axis=1)
+
+    algorithm = "PyMedPhys"
+    dataframe_by_algorithm = _filter_by(dataframe, "algorithm", algorithm)
+
+    # st.write(dataframe)
+
+    treatments = dataframe_by_algorithm["treatment"].unique()
+    # st.write(treatments)
+
+    treatment = treatments[0]
+    # st.write(treatment)
+
+    dataframe_by_treatment = _filter_by(dataframe_by_algorithm, "treatment", treatment)
```

Comment:
this is the dataframe filtered first by algorithm and then by treatment, but the name of the variable could be construed as dataframe by treatment only (no additional pre-filtering by algorithm).  letters are cheap.  time spent re-reading code is expensive.